scripts/release/update-svgs-count.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+/**
+ * @fileoverview
+ * Replaces the SVG count milestone "Over <NUMBER> Free SVG icons..." located
+ * at README every time the number of current icons is more than `updateRange`
+ * more than the previous milestone.
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const regexMatcher = /Over\s(\d+)\s/;
+const updateRange = 500;
+
+const rootDir = path.resolve(__dirname, '..', '..');
+const dataFile = path.resolve(rootDir, '_data', 'simple-icons.json');
+const readmeFile = path.resolve(rootDir, 'README.md');
+const readmeContent = fs.readFileSync(readmeFile).toString();
```

Comment:
Should add the `utf-8` encoding and remove `toString`

Reply:
Updated by https://github.com/simple-icons/simple-icons/pull/6951/commits/6d9fbfe3e5dc0617b0fd7ad717c1551bc1f55bf9 in other files as well